cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreServiceTestsSuite.java

```diff
@@ -31,12 +33,17 @@
   HiveExploreServiceTestRun.class,
   WritableDatasetTestRun.class,
   HiveExploreObjectMappedTableTestRun.class,
-  HiveExploreServiceStopTestRun.class
+  HiveExploreServiceFileSetTest.class
```

Comment:
should rename to HiveExploreServiceFileSetTestRun so that maven will not run it both in the suite and outside the suite